cafebabel/templates/users/detail.html

```diff
@@ -9,6 +9,8 @@ <h1>{{ user }}'s profile</h1>
   <h2>About</h2>
   {% if user.profile.has_image %}
   <img src={{ user.profile.image_url }} alt="Picture of {{ user }}">
+  {% else %}
+  <img src=/static/img/profile.svg width=200px height=200px alt="Blank profile picture">
```

Comment:
`Default` instead of `Blank`?

Reply:
Can't you set width/height using CSS?

Reply:
Not sure it works with _px_ anyways in HTML attributes

Reply:
oui